lib/internal/test_runner/test.js

```diff
@@ -146,7 +144,7 @@ class Test extends AsyncResource {
     }
 
     if (parent === null) {
-      this.concurrency = rootConcurrency;
+      this.concurrency = 1;
```

Comment:
?

Reply:
instead of testing for `isTestRunner` https://github.com/nodejs/node/blob/6ee1f3444f8c1cf005153f936ffc74221d55658b/lib/internal/test_runner/test.js#L61
I explicitly pass in this case `concurrecny: true`:
https://github.com/nodejs/node/blob/ae799bdf1a1c2576bcd1f17430bc8f5f1997a631/lib/internal/main/test_runner.js#L21
then it is set to the same value here:
https://github.com/nodejs/node/blob/6ee1f3444f8c1cf005153f936ffc74221d55658b/lib/internal/test_runner/test.js#L177-L180

it is the same flow but more explicit